packages/azure-http-specs/specs/azure/resource-manager/resources/mockapi.ts

```diff
@@ -151,16 +163,113 @@ const validResourceExtensionsResource = {
   },
 };
 
+function getServiceKey(resourceUri: string) {
+  switch (resourceUri) {
+    case RESOURCE_GROUP_SCOPE_URI:
+      return "ResourceGroup" as const;
+    case SUBSCRIPTION_SCOPE_URI:
+      return "Subscription" as const;
+    case TENANT_SCOPE_URI:
+      return "Tenant" as const;
+    case RESOURCE_SCOPE_URI:
+      return "Resource" as const;
+    default:
+      throw new ValidationError(
+        "Invalid resource uri",
+        `${RESOURCE_GROUP_SCOPE_URI} | ${SUBSCRIPTION_SCOPE_URI} | ${TENANT_SCOPE_URI} | ${RESOURCE_SCOPE_URI}`,
+        resourceUri,
+      );
+  }
+}
+
+function requestHandler(
+  req: MockRequest,
+  requestMethod: string,
+  resourceUri: string,
+  validResource: any,
+) {
+  const serviceKey = getServiceKey(resourceUri);
+  switch (requestMethod) {
+    case "get":
+      return {
+        pass: serviceKey,
+        status: 200,
+        body: json(validResource),
+      } as const;
+    case "put":
+      return {
+        pass: serviceKey,
+        status: 200,
+        body: json(validResource),
+      } as const;
+    case "patch":
+      return {
+        pass: serviceKey,
+        status: 200,
+        body: json({
+          ...validResource,
+          properties: {
+            provisioningState: "Succeeded",
+            description: "valid2",
+          },
+        }),
+      } as const;
+    case "delete":
+      return {
+        pass: serviceKey,
+        status: 204,
+      } as const;
+    case "list":
+      return {
+        pass: serviceKey,
+        status: 200,
+        body: json({
+          value: [validResource],
+        }),
+      } as const;
+    default:
+      throw new ValidationError(
+        "Invalid request method",
+        `"get" | "put" | "patch" | "delete" | "list"`,
+        requestMethod,
+      );
+  }
+}
+
 // extension tracked resource
-Scenarios.Azure_ResourceManager_Resources_ExtensionsResources_get = passOnSuccess([
+Scenarios.Azure_ResourceManager_Resources_ExtensionsResources_get = withServiceKeys([
+  "ResourceGroup",
+  "Subscription",
+  "Tenant",
+  "Resource",
+]).pass([
+  {
+    uri: "/:resourceUri/providers/Azure.ResourceManager.Resources/extensionsResources/:extensionName",
```

Comment:
or without "/" prefix?

Reply:
This version [bd42d17](https://github.com/Azure/typespec-azure/pull/2203/commits/bd42d17208014816dc40104885eff013266f1177) can check both "/" and "//".